mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -323,39 +338,40 @@ object CrossValidatorModel extends MLReadable[CrossValidatorModel] {
   @Since("1.6.0")
   override def load(path: String): CrossValidatorModel = super.load(path)
 
-  private[CrossValidatorModel]
+  /**
+   * Writer for CrossValidatorModel.
+   * @param instance CrossValidatorModel instance used to construct the writer
+   *
+   * Options:
+   * CrossValidatorModelWriter support an option "persistSubModels", available value is
```

Comment:
Fix wording:

CrossValidatorModelWriter supports an option "persistSubModels", with possible values "true" or "false". If you set the collectSubModels Param before fitting, then you can set "persistSubModels" to "true" in order to persist the submodels. By default, "persistSubModels" will be "true" when submodels are available and "false" otherwise.  If submodels are not available, then setting "persistSubModels" to "true" will cause an exception.